From dd8641c318500297d7ae166003d4e92143bc67a9 Mon Sep 17 00:00:00 2001
From: arvidn <arvid@libtorrent.org>
Date: Fri, 22 Jan 2021 02:53:48 +0100
Subject: [PATCH] fix version checking for using package.paths in Jamfile

---
 Jamfile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Jamfile b/Jamfile
index 1735ef2413..8eeff1145c 100644
--- a/Jamfile
+++ b/Jamfile
@@ -811,7 +811,7 @@ rule generate-pkg-config ( properties * )
 	# however, boost build's versioning scheme changed in boost-1.71 to version
 	# 4.0
 	local boost-build-version = [ SPLIT_BY_CHARACTERS [ version.boost-build ] : "-" ] ;
-	if [ version.version-less [ SPLIT_BY_CHARACTERS $(boost-build-version[1]) : "." ] : 2018 01 ]
+	if [ version.version-less [ SPLIT_BY_CHARACTERS $(boost-build-version[1]) : "." ] : 2018 03 ]
 	{
 		import option ;
 		import property ;
